### ISSUE TITLE
Compiling in separate build directory fails

### ISSUE THREAD

**Issue by boris-n** [the-tcpdump-group/libpcap#424]
When compiling libpcap 1.7.2 in a separate build directory, the build fails because scanner.c.top is not found. Fix is trivial: Simply replace "$@.top" by "$(srcdir)/$@.top" in Makefile.in.

**Comment by normjacobs** [comment]
I ran into this as well when building.  This fix is simple as suggested above.  The following  is a patch that embodies the suggested fix:

# https://github.com/the-tcpdump-group/libpcap/issues/424
```
--- libpcap-1.7.2/Makefile.in.orig      Wed Apr  8 10:55:32 2015
+++ libpcap-1.7.2/Makefile.in   Wed Apr  8 10:57:08 2015
@@ -447,7 +447,7 @@
        @rm -f $@ $@.bottom
        $(srcdir)/runlex.sh $(LEX) -o$@ $<
        mv $@ $@.bottom
-       cat $@.top $@.bottom > $@
+       cat $(srcdir)/$@.top $@.bottom > $@
        @rm $@.bottom

 scanner.o: scanner.c tokdefs.h
```

**Comment by guyharris** [comment]
Fixed in 1d62204e55be374c4ca10c49b4f704a79d93b527.
